client/src/components/photoCollage.jsx

```diff
@@ -2,24 +2,24 @@ import React from 'react';
 import styles from '../styles/photoCollage.css';
 
 
-const PhotoCollage = (data) => {
-  if (data.dataArr[6].photosAndComments.length < 5) {
+const PhotoCollage = (props) => {
+  if (props.dataArr.photosAndComments.length < 5) {
```

Comment:
Same as above, since the property names aren't as long we can take advantage of destructuring in the function argument line